assets/stylesheets/about.css

```diff
@@ -0,0 +1,11 @@
+article {
+  font-family: 'Raleway', Arial, Helvetica, sans-serif;
+}
+
+@media only screen and (min-width: 600px) {
```

Comment:
You're doing a media query here, but no alternative for smaller screens.  

Also you're repeating this in several pages, not DRY.